orm_lib/inc/drogon/orm/QueryBuilder.h

```diff
@@ -0,0 +1,56 @@
+/**
+ *
+ *  @file QueryBuilder.h
+ *  @author Ken Matsui
+ *
+ *  Copyright 2022, Ken Matsui.  All rights reserved.
+ *  https://github.com/drogonframework/drogon
+ *  Use of this source code is governed by a MIT license
+ *  that can be found in the License file.
+ *
+ *  Drogon
+ *
+ */
+
+#pragma once
+
+#include <drogon/orm/FilterBuilder.h>
+#include <string>
+
+namespace drogon
+{
+namespace orm
+{
+template <typename T>
+class QueryBuilder : public FilterBuilder<T, true>
+{
+    /**
+     * @brief When a user does not set the table name explicitly, then retrieve
+     * it from model `T`.
+     *
+     * @return std::string The table name
+     */
+    inline std::string getTableName() const
```

Comment:
this method could return a `const std::string &`

Reply:
Fixed :)